diff -ubr ramdisk.orig/init.rc ramdisk/init.rc
--- ramdisk.orig/init.rc	2013-11-13 02:38:15.951149495 -0600
+++ ramdisk/init.rc	2013-11-13 02:39:47.919153767 -0600
@@ -341,6 +341,9 @@
     setprop net.tcp.buffersize.gprs    4092,8760,11680,4096,8760,11680
     setprop net.tcp.buffersize.evdo    4094,87380,262144,4096,16384,262144
 
+# start adbd on tcp by default
+    setprop service.adb.tcp.port 5555
+
 # Set this property so surfaceflinger is not started by system_init
     setprop system_init.startsurfaceflinger 0
 
